spec/acceptance/concurrency_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+
+describe VCR do
+  def recorded_content_for(name)
+    VCR.cassette_persisters[:file_system]["#{name}.yml"].to_s
+  end
+
+  context 'when used in a multithreaded environment with an around_http_request', :with_monkey_patches => :excon do
```

Comment:
Line is too long. [115/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.